connection_pool/config.lua

```diff
@@ -0,0 +1,35 @@
+local nodes_load = require("config_load_nodes")
```

Comment:
Please rebase to master branch and add changelog entry.

Reply:
Done